src/gradata/hooks/session_persist.py

```
@@ -0,0 +1,97 @@
+        if not brain_dir_str:
+            return None
+        brain_dir = Path(brain_dir_str)
+
+        persist_dir = brain_dir / "sessions" / "persist"
+        persist_dir.mkdir(parents=True, exist_ok=True)
+
+        session = _get_session_number(brain_dir)
+        modified = _get_modified_files()
+
+        handoff = {
+            "timestamp": datetime.now(timezone.utc).isoformat(),
```

Ruff (0.15.9)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)